Open Judge System/Web/OJS.Web/Areas/Administration/Controllers/SubmissionsController.cs

```diff
@@ -116,6 +116,7 @@ public ActionResult Create(ModelType model)
                     entity.Processing = false;
                     this.BaseCreate(entity);
                     this.Data.SubmissionsForProcessing.AddOrUpdate(model.Id.Value);
+                    this.Data.SaveChanges();
```

Comment:
Put the creation of the submission and the adding to the new table in a transaction.